### PR TITLE
Replaces usage of User with Bidder

### PR DESCRIPTION
**Pull request by ckartik**
* Replaces the usage of user with bidder across the code.

### CODE REVIEW

**Comment by kant777** [comment]
Need to update this as well https://github.com/primevprotocol/mev-commit/blob/main/pkg/abi/abi.go#L1244 and run the integration test before merging this in

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
```
Comment (kant777) [inline README.md:3]: Need to update this as well https://github.com/primevprotocol/mev-commit/blob/main/pkg/abi/abi.go#L1244 and run the integration test before merging this in

**Review by kant777 — COMMENTED** [review]
Need to update this as well https://github.com/primevprotocol/mev-commit/blob/main/pkg/abi/abi.go#L1244 and run the integration test before merging this in

**Review by kant777 — APPROVED** [review]

File: contracts/BidderRegistry.sol [hunk 2]
```diff
@@ -31,20 +31,20 @@ contract UserRegistry is IUserRegistry, Ownable, ReentrancyGuard {
...
 
-    /// @dev Mapping for if user is registered
-    mapping(address => bool) public userRegistered;
+    /// @dev Mapping for if bidder is registered
+    mapping(address => bool) public bidderRegistered;
 
-    /// @dev Mapping from user addresses to their staked amount
-    mapping(address => uint256) public userStakes;
+    /// @dev Mapping from bidder addresses to their staked amount
+    mapping(address => uint256) public bidderStakes;
```
[the remark this answers is not in the document]
Comment (ckartik) [reply contracts/BidderRegistry.sol:38]: That makes sense

**Pull request merged**